### DISCUSSION TITLE
packagist.org submit packages for first time users - comprehensive documentation around?

### DISCUSSION THREAD

**Discussion by siims-biz** [composer/composer#11423] [Q&A / Support]
While a lot of people already understand how composer/packagist/github works - there is always the odd one around who does not know.
I was searching a bit on the internet about topics like.
1) github/vendor/repository_packagist_hello_world 
2) Submitting that package to packagist 
3) Using that package in github/vendor/repository_composer_hello_world
4) Extending the package from 1)
5) Using that extension in 3)
Some third parties are writing about stuff like that.
At composer/packagist I did not spot a comprehensive document for first time users.
Perhaps something like
https://github.com/composer/packagist/wiki
could get used for that purpose.
Any thoughts.

**Comment by Seldaek** [comment]
There is https://getcomposer.org/doc/00-intro.md for using, and https://packagist.org/ for publishing. IMO that's enough and it seems to be for most. There are third parties blog posts or video tutorials for people looking for more I guess, so I don't really think we need to carry the full burden of documenting this further (or maintaining a wiki)
